microsoft-azure-sb/3.0/modules/ROOT/pages/azure-service-bus-connector-examples.adoc

```diff
@@ -0,0 +1,361 @@
+= Azure Service Bus Connector 3.0 Examples - Mule 4
+
+The examples in this section show application flows that:
+
+* <<send-receive,Sends messages to and receives messages>> from Azure
+* <<send-listen-ack, Sends and listens for messages and receives an acknowledgment using the `MANUAL` acknowledgment type
+
+These examples use variables for some field values. You can either:
+
+* Replace the variables with their values in the code
+* Provide the values for each variable in a properties file and then refer to that file from the connector configuration
++
+For more information about using a properties file, see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders]
+
+== Prerequisites
+
+* Java 8.
+* Anypoint Studio 7.1.0 or later
+* Mule Runtime 4.2.x EE or later
+* Create a new queue for testing purpose on Azure Portal.
+
+== Create a Mule Project
+
+After you create a new Mule project and configure your credentials, you can create an application flow for each example by using the provided XML code.
+
+To create a Mule project:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+. Create a file named `mule-app.properties` and save it in `<project-name>/src/main/resources`.
+. Configure your Azure connection credentials in the file `mule-app.properties`, and include the name of the Azure queue you created, for example:
++
+----
+#==============================================================
+#Connection Parameters
+#==============================================================
+# Azure
+azure.namespace=myazure
+azure.keyname=SAStokenname
+azure.key=SAStoken
+queue.name=mytestqueue
+----
++ 
+. Save the file.
+. Open the *Global Element Configuration* and click *Test Connection* to verify your credentials.
+
+[[send-receive]]
+== Example Send and Receive Flow
+
+This flow shows an example of how to use the Send and Receive From Queue operations to  send and receive messages:
```

Comment:
The operations should be in backtick. It looks like there might be an extra space after "to."